SayTheirNames/Source/Dependency/DependencyContainer.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  Services.swift
+//  SayTheirNames
+//
+//  Copyright (c) 2020 Say Their Names Team (https://github.com/Say-Their-Name)
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+
+import UIKit
+
+// MARK: - Dependency
+protocol Dependency {
+    
+}
+
+// MARK: - DependencyContainer
+final class DependencyContainer: Dependency {
+    let navigator = Navigator()
+    let image = ImageService()
+    let dateFormatter = DateFormatterService()
+    let network = NetworkRequestor()
+    
+    // MARK: - Init
+    init() {
```

Comment:
Would it be possible to pass in Navigator, ImageService, etc. in the initializer instead of instantiating them as properties? It's more of a preference thing, but it makes it a little less bound to those services and adds a little more visibility at the DependencyContainer call site. Also, if things like ImageService become protocols it'll make it way easier to swap in testing components for tests and alternative implementations.

Reply:
FWIW it doesn't have to happen on this pull-request, but just a suggestion for the future :)